ui/src/store/modules/permission.js

```diff
@@ -17,24 +17,39 @@
 
 import { asyncRouterMap, constantRouterMap } from '@/config/router'
 
-function hasApi (apis, route) {
-  if (route.meta && route.meta.permission) {
-    for (const permission of route.meta.permission) {
-      if (!apis.includes(permission)) {
-        return false
-      }
-    }
+function hasAccessToRoute (apis, route) {
+  if (!route.meta || !route.meta.permission) {
```

Comment:
Could use optional chaining here, no?
```suggestion
  if (!route.meta?.permission) {
```

Reply:
Never mind, did not see it was already merged.